notebooks/week2/04_log_and_register_custom_model.py

```
@@ -0,0 +1,123 @@
+# Databricks notebook source
+import mlflow
+import numpy as np
```

Ruff

`numpy` imported but unused

Remove unused import: `numpy`

(F401)